src/components/general/Blog.vue

```diff
@@ -9,12 +25,122 @@ export default {
 	props: {
 		limit: Number
 	},
+	data() {
+		return {
+			blogData: []
+		}
+	},
+	methods: {
+		toText(htmlText) {
+			let plainText = document.createElement('div')
+			plainText.innerHTML = htmlText
+			htmlText = plainText.innerText
+			return htmlText
+		}
+	},
+	beforeMount() {
+		let referenceThis = this
+		let httpRequest = new XMLHttpRequest();
+		httpRequest.onreadystatechange = function() {
+			if (this.readyState == 4 && this.status == 200) {
+				referenceThis.blogData = JSON.parse(this.responseText);
+			}
+		};
+		httpRequest.open("GET", "http://test.cutc.ca/mediumPosts.php", true);
```

Comment:
Temporary URL to be changed to `api.cutc.ca/mediumPosts.php` before deploying to production.